href.js

```diff
@@ -8,15 +8,15 @@ const noRoutingAttrName = 'data-no-routing'
 // handle a click if is anchor tag with an href
 // and url lives on the same domain. Replaces
 // trailing '#' so empty links work as expected.
-// fn(str) -> null
-function href (cb) {
+// (fn(str), obj?) -> undefined
```

Comment:
could you leave this as null? thanks - I know it's only a convention and not enforced so I know it's a bit silly but yeah I use `null` like C's `void` return. Thank you!